integration/batch-modular-default/run.py

```diff
@@ -31,6 +31,9 @@
 )
 for logger in loggers_to_mute:
     logging.getLogger(logger).setLevel(logging.ERROR)
+# The concurrency logs may be different for every machine, so let's
+# just turn them off
+logging.getLogger("klio.concurrency").setLevel(logging.ERROR)
```

Comment:
Just wondering if there is a reason `"klio.concurrency"` is turned off explicitly rather than added to `loggers_to_mute`.

Reply:
ah...tbh I didn't even think to put them in `loggers_to_mute` 😆 good idea